data/registry/instrumentation-java-elasticsearch-client.yml

```diff
@@ -18,3 +18,4 @@ urls:
   docs: https://www.elastic.co/guide/en/elasticsearch/client/java-api-client/current/opentelemetry.html
 createdAt: '2024-08-06'
 isNative: true
+isFirstParty: true
```

Comment:
```suggestion
```

Please do not include the "isFirstParty" flag when "isNative" is set, this will most likely to let unexpected behavior in the UI (showing a first party and a native badge for the instrumentation library)